Yoast/javascript
# Add specs for absolute superlative suffixes

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds specs to test the stemming of absolute superlative suffixes (-issimo/a/e/i) in Italian

## Test instructions

This PR can be tested by following these steps:

* Run yarn test and make sure all current specs are passing, and that the coverage of the stemmer is at 100 %.
* Add more words with absolute superlative suffixes (-issimX, -atissimX, -itissimX, -issimamente) to the specs and make sure they still pass.
* In Italian Premium: Add forms of adjectives/adverbs, including absolute superlative forms, to the focus keyphrase and to the title/text/meta description and make sure they are recognized. For example, _forte_, _fortissimo_, and _fortissimamente_ should all be recognised as the same word.


## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-422
